cpp/oneapi/dal/algo/rbf_kernel/backend/gpu/compute_kernel_dense_dpc.cpp

```diff
@@ -29,46 +27,113 @@ using input_t = compute_input<task::compute>;
 using result_t = compute_result<task::compute>;
 using descriptor_t = detail::descriptor_base<task::compute>;
 
-namespace daal_rbf_kernel = daal::algorithms::kernel_function::rbf;
-namespace interop = dal::backend::interop;
+namespace pr = dal::backend::primitives;
+
+template <typename Float>
+inline auto compute_rbf_values(sycl::queue& queue,
+                               const pr::ndview<Float, 1>& sqr_x_nd,
+                               const pr::ndview<Float, 1>& sqr_y_nd,
+                               pr::ndview<Float, 2>& res_nd,
+                               const Float coeff,
```

Comment:
coeff is used in this function only - there is no need to declare it in `compute_rbf()` and pass it as a parameter to this function, you can declare it here